lib/mhv_ac/registration_form.rb

```diff
@@ -33,9 +33,11 @@ class RegistrationForm < Common::Form
     attribute :terms_version, String
     attribute :terms_accepted_date, Common::HTTPDate
 
-    validates :icn, :is_patient, :is_veteran, :email, presence: true
-    validates :address1, :city, :state, :zip, :country, presence: true
-    validates :sign_in_partners, :terms_version, :terms_accepted_date, presence: true
+    # NOTE: commenting out these validations because it's not entirely clear they're required
+    # instead for now we're going to rely on MHV kicking back its error based on these.
+    # validates :icn, :is_patient, :is_veteran, :email, presence: true
```

Comment:
yea, so if they're required by MHV they will raise an error. I don't know what good it does for us pre-emptively raising that error unless theirs are just completely nonsensical. If it results in a BackendServiceException we can always handle it. I think these type of validations are particularly useful if you want to validate user input, maybe a little less so if everything is derived by vets.gov on the backend.